src/server/req_quejob.c

```diff
@@ -1808,11 +1810,11 @@ req_commit(struct batch_request *preq)
 
 	delete_link(&pj->ji_alljobs);
 	append_link(&svr_alljobs, &pj->ji_alljobs, pj);
+	tree_add_del(AVL_jobs, (void *)pj->ji_qs.ji_jobid, (void *)pj, TREE_OP_ADD);
```

Comment:
Job's avl tree is not new, can you point me to where we used to add jobs to this avl tree before your change?

Reply:
This is the mom part, earlier we used AVL tree for jobs only in Server code. For server code we do AVL operation for jobs in svr_enqueue() and svr_dequeue() functions